--- openh323/src/speexcodec.cxx~	2004-01-30 01:55:41.000000000 +0100
+++ openh323/src/speexcodec.cxx	2004-03-19 09:32:20.447009240 +0100
@@ -394,7 +394,7 @@
 BOOL SpeexCodec::EncodeFrame(BYTE * buffer, unsigned & length)
 {
   // convert PCM to float
-  float floatData[SAMPLES_PER_FRAME];
+  short floatData[SAMPLES_PER_FRAME];
   PINDEX i;
   for (i = 0; i < SAMPLES_PER_FRAME; i++)
     floatData[i] = sampleBuffer[i];
@@ -411,7 +411,7 @@
 
 BOOL SpeexCodec::DecodeFrame(const BYTE * buffer, unsigned length, unsigned &)
 {
-  float floatData[SAMPLES_PER_FRAME];
+  short floatData[SAMPLES_PER_FRAME];
 
   // decode Speex data to floats
   speex_bits_read_from(bits, (char *)buffer, length); 
