diff -u -r openh323.orig/configure.in openh323/configure.in
--- openh323.orig/configure.in	2004-01-02 02:23:12.000000000 +0100
+++ openh323/configure.in	2004-01-26 06:02:29.000000000 +0100
@@ -79,12 +79,6 @@
   PWLIBDIR=`$PTLIB_CONFIG --prefix`
 fi
 
-if test "x$PWLIBDIR" = "x/usr" -o "x$PWLIBDIR" = "x/usr/"; then
-  PWLIBDIR="/usr/share/pwlib/"
-fi
-if test "xPWLIBDIR" = "x/usr/local" -o "x$PWLIBDIR" = "x/usr/"; then
-  PWLIBDIR="/usr/local/share/pwlib/"
-fi
 
 echo "PWLib prefix set to.... $PWLIBDIR"
 
diff -u -r openh323.orig/openh323u.mak.in openh323/openh323u.mak.in
--- openh323.orig/openh323u.mak.in	2004-01-02 02:23:13.000000000 +0100
+++ openh323/openh323u.mak.in	2004-01-26 06:03:49.000000000 +0100
@@ -135,6 +135,7 @@
 #
 
 PWLIBDIR	= @PWLIBDIR@
+PWLIBMAKEDIR  = @PWLIBDIR@/share/pwlib/make
 STDCCFLAGS	+= @STDCCFLAGS@
 LDFLAGS		+= @LDFLAGS@
 ENDLDLIBS	:= @ENDLDLIBS@ $(ENDLDLIBS)
@@ -144,12 +145,12 @@
 H323_SYSTEM_SPEEX = @H323_SYSTEM_SPEEX@
 
 ifdef LIBRARY_MAKEFILE
-include $(PWLIBDIR)/make/unix.mak
+include $(PWLIBMAKEDIR)/unix.mak
 else
 ifdef NOTRACE
 OBJDIR_SUFFIX := n
 endif
-include $(PWLIBDIR)/make/ptlib.mak
+include $(PWLIBMAKEDIR)/ptlib.mak
 endif
 
 LIBDIRS += $(OPENH323DIR)
@@ -181,7 +182,7 @@
 STDCCFLAGS += -DNO_H323_VIDEO
 endif
 
-OH323_BASE  = h323_$(PLATFORM_TYPE)_$(OH323_SUFFIX)$(LIB_TYPE)
+OH323_BASE  = openh323
 OH323_FILE  = lib$(OH323_BASE).$(LIB_SUFFIX)
 
 LDFLAGS	    += -L$(OH323_LIBDIR)
@@ -289,6 +290,7 @@
 
 $(TARGET) :	$(OH323_LIBDIR)/$(OH323_FILE)
 
+ifdef OPENH323_BUILD
 ifndef LIBRARY_MAKEFILE
 ifdef DEBUG
 $(OH323_LIBDIR)/$(OH323_FILE):
@@ -298,6 +300,7 @@
 	$(MAKE) -C $(OH323_SRCDIR) opt
 endif
 endif
+endif
 
 # End of file
 
diff -u -r openh323.orig/src/Makefile openh323/src/Makefile
--- openh323.orig/src/Makefile	2003-12-29 13:13:26.000000000 +0100
+++ openh323/src/Makefile	2004-01-26 06:02:29.000000000 +0100
@@ -843,7 +843,7 @@
 
 ####################################################
 
-include $(PWLIBDIR)/make/common.mak
+include $(PWLIBMAKEDIR)/common.mak
 
 ####################################################
 
@@ -855,7 +855,7 @@
 LIB_BASENAME=$(OH323_BASE)
 LIB_FILENAME=$(OH323_FILE)
 
-include $(PWLIBDIR)/make/lib.mak
+include $(PWLIBMAKEDIR)/lib.mak
 
 ####################################################
 
