diff -urN openh323.orig/openh323u.mak openh323/openh323u.mak
--- openh323.orig/openh323u.mak	Wed Sep 18 12:48:00 2002
+++ openh323/openh323u.mak	Wed Sep 18 13:01:29 2002
@@ -101,12 +101,12 @@
 
 
 ifdef LIBRARY_MAKEFILE
-include $(PWLIBDIR)/make/unix.mak
+include $(PWLIBDIR)/share/pwlib/unix.mak
 else
 ifdef NOTRACE
 OBJDIR_SUFFIX := n
 endif
-include $(PWLIBDIR)/make/ptlib.mak
+include $(PWLIBDIR)/share/pwlib/ptlib.mak
 endif
 
 
@@ -137,7 +137,7 @@
 
 
 
-OH323_BASE  = h323_$(PLATFORM_TYPE)_$(OH323_SUFFIX)$(LIB_TYPE)
+OH323_BASE  = openh323
 OH323_FILE  = lib$(OH323_BASE).$(LIB_SUFFIX)
 
 LDFLAGS	    += -L$(OH323_LIBDIR)
@@ -213,6 +213,7 @@
 
 $(TARGET) :	$(OH323_LIBDIR)/$(OH323_FILE)
 
+ifdef OPENH323_BUILD
 ifndef LIBRARY_MAKEFILE
 ifdef DEBUG
 $(OH323_LIBDIR)/$(OH323_FILE):
@@ -222,7 +223,7 @@
 	$(MAKE) -C $(OH323_SRCDIR) opt
 endif
 endif
-
+endif
 
 # End of file
 
diff -urN openh323.orig/src/Makefile openh323/src/Makefile
--- openh323.orig/src/Makefile	Wed Sep 18 12:47:59 2002
+++ openh323/src/Makefile	Wed Sep 18 12:53:45 2002
@@ -611,7 +611,7 @@
 
 ####################################################
 
-include $(PWLIBDIR)/make/common.mak
+include $(PWLIBDIR)/share/pwlib/common.mak
 
 ####################################################
 
@@ -624,7 +624,7 @@
 LIB_BASENAME=$(OH323_BASE)
 LIB_FILENAME=$(OH323_FILE)
 
-include $(PWLIBDIR)/make/lib.mak
+include $(PWLIBDIR)/share/pwlib/lib.mak
 
 ####################################################
 
