diff -Naur openh323_v1_17_1.orig/configure.ac openh323_v1_17_1/configure.ac
--- openh323_v1_17_1.orig/configure.ac	2005-05-18 18:27:25.000000000 +0200
+++ openh323_v1_17_1/configure.ac	2005-05-18 18:27:18.000000000 +0200
@@ -75,13 +75,6 @@
   PWLIBDIR=`$PTLIB_CONFIG --prefix`
 fi
 
-if test "x$PWLIBDIR" = "x/usr" -o "x$PWLIBDIR" = "x/usr/"; then
-  PWLIBDIR="/usr/share/pwlib/"
-fi
-if test "x$PWLIBDIR" = "x/usr/local" -o "x$PWLIBDIR" = "x/usr/"; then
-  PWLIBDIR="/usr/local/share/pwlib/"
-fi
-
 echo "PWLib prefix set to.... $PWLIBDIR"
 
 AC_SUBST(PWLIBDIR)
diff -Naur openh323_v1_17_1.orig/Makefile.in openh323_v1_17_1/Makefile.in
--- openh323_v1_17_1.orig/Makefile.in	2004-11-27 19:42:15.000000000 +0100
+++ openh323_v1_17_1/Makefile.in	2005-05-18 18:27:18.000000000 +0200
@@ -331,15 +331,13 @@
 
 install:
 	mkdir -p $(DESTDIR)$(LIBDIR)
-	rm -f $(DESTDIR)$(LIBDIR)/libh323_*_r.so*
-	$(INSTALL) -m 444 lib/*_r.so $(DESTDIR)$(LIBDIR)
-	(for fn in lib/*_r.so.* ; do \
+	rm -f $(DESTDIR)$(LIBDIR)/libopenh323*.so*
+	$(INSTALL) lib/*.so.*.*.* $(DESTDIR)$(LIBDIR)
+	(for fn in lib/*.so ; do \
 		cd $(DESTDIR)$(LIBDIR) ; \
 		rm -f `basename $$fn` ; \
-		ln -s libh323_*_r.so `basename $$fn` ; \
+		ln -s libopenh323.so.*.*.* `basename $$fn` ; \
 	done)
-	$(INSTALL) -m 444 lib/$(OH323_FILE) $(DESTDIR)$(LIBDIR)
-	( cd $(DESTDIR)$(LIBDIR); rm libopenh323.so ; ln -s $(OH323_FILE) libopenh323.so)
 	( for dir in $(DESTDIR)$(LIBDIR) \
 		     $(DESTDIR)$(PREFIX)/include/openh323 \
 		     $(DESTDIR)$(PREFIX)/share/openh323 ;\
diff -Naur openh323_v1_17_1.orig/openh323u.mak.in openh323_v1_17_1/openh323u.mak.in
--- openh323_v1_17_1.orig/openh323u.mak.in	2005-05-18 18:27:25.000000000 +0200
+++ openh323_v1_17_1/openh323u.mak.in	2005-05-18 18:27:18.000000000 +0200
@@ -163,9 +163,11 @@
 #
 
 PWLIBDIR	= @PWLIBDIR@
-OPENH323DIR	= @OPENH323DIR@
+PWLIBMAKEDIR    = @PWLIBDIR@/share/pwlib/make
+ifndef OPENH323DIR
+OPENH323DIR     = /usr
+endif
 STDCCFLAGS	+= @STDCCFLAGS@
-LDFLAGS		+= @LDFLAGS@
 LDLIBS		+= @LDLIBS@
 ENDLDLIBS	:= @ENDLDLIBS@ $(ENDLDLIBS)
 
@@ -184,12 +186,12 @@
 H323_IXJ	     = @H323_IXJ@
 
 ifdef LIBRARY_MAKEFILE
-include $(PWLIBDIR)/make/unix.mak
+include $(PWLIBMAKEDIR)/unix.mak
 else
 ifeq ($(NOTRACE), 1)
 OBJDIR_SUFFIX := n
 endif # NOTRACE
-include $(PWLIBDIR)/make/ptlib.mak
+include $(PWLIBMAKEDIR)/ptlib.mak
 endif # LIBRARY_MAKEFILE
 
 LIBDIRS += $(OPENH323DIR)
@@ -204,7 +206,7 @@
 endif # PREFIX
 
 ifndef OH323_LIBDIR
-OH323_LIBDIR = $(OPENH323DIR)/lib
+OH323_LIBDIR = $(subst $(PWLIBDIR),$(OPENH323DIR),$(PW_LIBDIR))
 endif # OH323_LIBDIR
 
 ifeq ($(NOTRACE), 1)
@@ -216,7 +218,7 @@
 OH323_SUFFIX = $(OBJ_SUFFIX)
 endif # NOTRACE
 
-OH323_BASE  = h323_$(PLATFORM_TYPE)_$(OH323_SUFFIX)$(LIB_TYPE)
+OH323_BASE  = openh323
 OH323_FILE  = lib$(OH323_BASE).$(LIB_SUFFIX)
 
 LDFLAGS	    := -L$(OH323_LIBDIR) $(LDFLAGS)
@@ -271,6 +273,7 @@
 
 $(TARGET) :	$(OH323_LIBDIR)/$(OH323_FILE)
 
+ifdef OPENH323_BUILD
 ifndef LIBRARY_MAKEFILE
 ifdef DEBUG
 $(OH323_LIBDIR)/$(OH323_FILE):
@@ -280,6 +283,7 @@
 	$(MAKE) -C $(OH323_SRCDIR) opt
 endif # DEBUG
 endif # LIBRARY_MAKEFILE
+endif # OPENH323_BUILD
 
 # End of file
 
diff -Naur openh323_v1_17_1.orig/src/Makefile openh323_v1_17_1/src/Makefile
--- openh323_v1_17_1.orig/src/Makefile	2005-05-18 18:27:25.000000000 +0200
+++ openh323_v1_17_1/src/Makefile	2005-05-18 18:27:18.000000000 +0200
@@ -376,7 +376,7 @@
 
 ####################################################
 
-include $(PWLIBDIR)/make/common.mak
+include $(PWLIBMAKEDIR)/common.mak
 
 ####################################################
 
@@ -388,7 +388,7 @@
 LIB_BASENAME=$(OH323_BASE)
 LIB_FILENAME=$(OH323_FILE)
 
-include $(PWLIBDIR)/make/lib.mak
+include $(PWLIBMAKEDIR)/lib.mak
 
 ####################################################
 
