diff -Naur openh323-1.15.3.orig/configure.ac openh323-1.15.3/configure.ac
--- openh323-1.15.3.orig/configure.ac	2005-05-20 11:08:00.000000000 +0200
+++ openh323-1.15.3/configure.ac	2005-05-20 11:08:55.000000000 +0200
@@ -200,14 +200,14 @@
     fi
 
     if test "${H323_AVCODEC:-unset}" != "unset" ; then
-      AC_CHECK_FILE(/usr/local/include/ffmpeg/avcodec.h, H323_AVCODEC=1)
+      AC_CHECK_HEADER([ffmpeg/avcodec.h], H323_AVCODEC=1)
     fi
 
     if test "${H323_AVCODEC:-unset}" != "unset" ; then
       AC_SUBST(H323_AVCODEC, 1)
       AC_DEFINE(H323_AVCODEC, 1, "ffmpeg based avodec found")
-      AVCODECINCLUDEDIR=/usr/local/include/ffmpeg
-      AVCODECLIBDIR=/usr/local/lib
+      AVCODECINCLUDEDIR=/usr/include
+      AVCODECLIBDIR=/usr/lib
       STDCCFLAGS="$STDCCFLAGS -I${AVCODECINCLUDEDIR}"
 
       echo  "                      H263 codec on FFmpeg library  enabled"
diff -Naur openh323-1.15.3.orig/src/ffh263codec.cxx openh323-1.15.3/src/ffh263codec.cxx
--- openh323-1.15.3.orig/src/ffh263codec.cxx	2005-05-20 11:08:00.000000000 +0200
+++ openh323-1.15.3/src/ffh263codec.cxx	2005-05-20 11:08:55.000000000 +0200
@@ -164,7 +164,7 @@
 #define  MSVC_OPENH323 1
 
 extern "C" {
-#include <avcodec.h>
+#include <ffmpeg/avcodec.h>
 };
 
 
