diff -urN openh323.org/configure.in openh323/configure.in
--- openh323.org/configure.in	2004-03-18 01:01:15.115222776 +0100
+++ openh323/configure.in	2004-03-18 01:01:22.654076696 +0100
@@ -83,12 +83,6 @@
   PWLIBDIR=`$PTLIB_CONFIG --prefix`
 fi
 
-if test "x$PWLIBDIR" = "x/usr" -o "x$PWLIBDIR" = "x/usr/"; then
-  PWLIBDIR="/usr/share/pwlib/"
-fi
-if test "xPWLIBDIR" = "x/usr/local" -o "x$PWLIBDIR" = "x/usr/"; then
-  PWLIBDIR="/usr/local/share/pwlib/"
-fi
 
 echo "PWLib prefix set to.... $PWLIBDIR"
 
diff -urN openh323.org/openh323u.mak.in openh323/openh323u.mak.in
--- openh323.org/openh323u.mak.in	2004-03-18 01:01:15.135219736 +0100
+++ openh323/openh323u.mak.in	2004-03-18 01:02:03.699836792 +0100
@@ -141,6 +141,9 @@
 #
 
 PWLIBDIR	= @PWLIBDIR@
-OPENH323DIR	= @OPENH323DIR@
+PWLIBMAKEDIR	= @PWLIBDIR@/share/pwlib/make
+ifndef OPENH323DIR
+OPENH323DIR	= /usr
+endif
 STDCCFLAGS	+= @STDCCFLAGS@
 LDFLAGS		+= @LDFLAGS@
@@ -151,12 +152,12 @@
 H323_SYSTEM_SPEEX = @H323_SYSTEM_SPEEX@
 
 ifdef LIBRARY_MAKEFILE
-include $(PWLIBDIR)/make/unix.mak
+include $(PWLIBMAKEDIR)/unix.mak
 else
 ifdef NOTRACE
 OBJDIR_SUFFIX := n
 endif
-include $(PWLIBDIR)/make/ptlib.mak
+include $(PWLIBMAKEDIR)/ptlib.mak
 endif
 
 LIBDIRS += $(OPENH323DIR)
@@ -190,7 +191,7 @@
 STDCCFLAGS += -DNO_H323_VIDEO
 endif
 
-OH323_BASE  = h323_$(PLATFORM_TYPE)_$(OH323_SUFFIX)$(LIB_TYPE)
+OH323_BASE  = openh323
 OH323_FILE  = lib$(OH323_BASE).$(LIB_SUFFIX)
 
 LDFLAGS	    += -L$(OH323_LIBDIR)
@@ -283,6 +284,7 @@
 
 $(TARGET) :	$(OH323_LIBDIR)/$(OH323_FILE)
 
+ifdef OPENH323_BUILD
 ifndef LIBRARY_MAKEFILE
 ifdef DEBUG
 $(OH323_LIBDIR)/$(OH323_FILE):
@@ -292,6 +294,7 @@
 	$(MAKE) -C $(OH323_SRCDIR) opt
 endif
 endif
+endif
 
 # End of file
 
diff -urN openh323.org/src/Makefile openh323/src/Makefile
--- openh323.org/src/Makefile	2004-03-18 01:01:15.309193288 +0100
+++ openh323/src/Makefile	2004-03-18 01:01:22.636079432 +0100
@@ -847,7 +847,7 @@
 
 ####################################################
 
-include $(PWLIBDIR)/make/common.mak
+include $(PWLIBMAKEDIR)/common.mak
 
 ####################################################
 
@@ -859,7 +859,7 @@
 LIB_BASENAME=$(OH323_BASE)
 LIB_FILENAME=$(OH323_FILE)
 
-include $(PWLIBDIR)/make/lib.mak
+include $(PWLIBMAKEDIR)/lib.mak
 
 ####################################################
 
