diff -urN openh323.org/Makefile openh323/Makefile
--- openh323.org/Makefile	Tue Jan  8 02:24:42 2002
+++ openh323/Makefile	Tue Jan  8 02:25:04 2002
@@ -234,7 +234,7 @@
 	$(MAKE) all
 
 
-include $(PWLIBDIR)/make/ptlib.mak
+include $(PWLIBDIR)/share/pwlib/ptlib.mak
 
 
 # Pass through commands to the subdirectory for sample app
diff -urN openh323.org/openh323u.mak openh323/openh323u.mak
--- openh323.org/openh323u.mak	Tue Jan  8 02:24:42 2002
+++ openh323/openh323u.mak	Tue Jan  8 02:25:05 2002
@@ -71,7 +71,7 @@
 OBJDIR_SUFFIX := n
 endif
 endif
-include $(PWLIBDIR)/make/ptlib.mak
+include $(PWLIBDIR)/share/pwlib/ptlib.mak
 endif
 
 
@@ -102,7 +102,7 @@
 
 
 
-OH323_BASE  = h323_$(PLATFORM_TYPE)_$(OH323_SUFFIX)$(LIB_TYPE)
+OH323_BASE  = openh323
 OH323_FILE  = lib$(OH323_BASE).$(LIB_SUFFIX)
 
 LDFLAGS	    += -L$(OH323_LIBDIR)
@@ -165,6 +165,7 @@
 
 $(TARGET) :	$(OH323_LIBDIR)/$(OH323_FILE)
 
+ifdef OPENH323_BUILD
 ifndef OH323_LIBRARY_MAKEFILE
 ifdef DEBUG
 $(OH323_LIBDIR)/$(OH323_FILE):
@@ -174,7 +175,7 @@
 	$(MAKE) -C $(OH323_SRCDIR) opt
 endif
 endif
-
+endif
 
 # End of file
 
diff -urN openh323.org/src/Makefile openh323/src/Makefile
--- openh323.org/src/Makefile	Tue Jan  8 02:24:43 2002
+++ openh323/src/Makefile	Tue Jan  8 02:25:05 2002
@@ -497,7 +497,7 @@
 
 ####################################################
 
-include $(PWLIBDIR)/make/common.mak
+include $(PWLIBDIR)/share/pwlib/common.mak
 
 ####################################################
 
@@ -510,7 +510,7 @@
 LIB_BASENAME=$(OH323_BASE)
 LIB_FILENAME=$(OH323_FILE)
 
-include $(PWLIBDIR)/make/lib.mak
+include $(PWLIBDIR)/share/pwlib/lib.mak
 
 ####################################################
 
