--- openh323-1.15.3/plugins/audio/GSM0610/gsm06_10_codec.c.orig	2005-02-07 01:41:31.000000000 +0100
+++ openh323-1.15.3/plugins/audio/GSM0610/gsm06_10_codec.c	2005-03-25 22:21:12.020016968 +0100
@@ -76,7 +76,7 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include "inc/gsm.h"
+#include <gsm.h>
 
 #define	BITS_PER_SECOND         13200
 
--- openh323-1.15.3/plugins/audio/GSM0610/Makefile.in.orig	2004-08-24 16:15:12.000000000 +0200
+++ openh323-1.15.3/plugins/audio/GSM0610/Makefile.in	2005-03-25 22:22:39.295749040 +0100
@@ -60,7 +60,7 @@
 PREFIX=@INSTALLPREFIX@
 exec_prefix = ${PREFIX}
 LIBDIR=@LIBDIR@
-H323_SYSTEM_GSM=@H323_SYSTEM_GSM@
+H323_SYSTEM_GSM=1
 H323_EMBEDDED_GSM=@H323_EMBEDDED_GSM@
 
 CC=@CC@
@@ -79,7 +79,7 @@
 
 SRCS	+= gsm06_10_codec.c
 
-#ifeq ($(H323_SYSTEM_GSM),0)
+ifeq ($(H323_SYSTEM_GSM),0)
 
 SRCS	+= $(SRCDIR)/gsm_create.c \
            $(SRCDIR)/gsm_destroy.c \
@@ -98,11 +98,11 @@
 
 EXTRALIBS = 
 
-#else
-#
-#EXTRALIBS = -lgsm
-#
-#endif
+else
+
+EXTRALIBS = -lgsm
+
+endif
 
 vpath	%.o $(OBJDIR)
 vpath	%.c $(SRCDIR)
--- openh323-1.15.3/plugins/audio/LPC_10/lpc_10_codec.c.orig	2005-02-07 01:41:32.000000000 +0100
+++ openh323-1.15.3/plugins/audio/LPC_10/lpc_10_codec.c	2005-03-25 22:21:27.805617192 +0100
@@ -41,7 +41,7 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include "lpc10.h"
+#include <lpc10.h>
 
 PLUGIN_CODEC_IMPLEMENT("LPC 10")
 
--- openh323-1.15.3/plugins/audio/LPC_10/Makefile.in.orig	2004-08-22 06:20:19.000000000 +0200
+++ openh323-1.15.3/plugins/audio/LPC_10/Makefile.in	2005-03-25 22:23:44.907774496 +0100
@@ -52,41 +52,8 @@
 
 HEADER_FILES	+= $(LPC10_INCDIR)/lpc10.h
 
-SRCS	+= lpc_10_codec.c \
-	   $(SRCDIR)/f2clib.c \
-	   $(SRCDIR)/analys.c \
-	   $(SRCDIR)/bsynz.c \
-	   $(SRCDIR)/chanwr.c \
-	   $(SRCDIR)/dcbias.c \
-	   $(SRCDIR)/decode_.c \
-	   $(SRCDIR)/deemp.c \
-	   $(SRCDIR)/difmag.c \
-	   $(SRCDIR)/dyptrk.c \
-	   $(SRCDIR)/encode_.c \
-	   $(SRCDIR)/energy.c \
-	   $(SRCDIR)/ham84.c \
-	   $(SRCDIR)/hp100.c \
-	   $(SRCDIR)/invert.c \
-	   $(SRCDIR)/irc2pc.c \
-	   $(SRCDIR)/ivfilt.c \
-	   $(SRCDIR)/lpcdec.c \
-	   $(SRCDIR)/lpcenc.c \
-	   $(SRCDIR)/lpcini.c \
-	   $(SRCDIR)/lpfilt.c \
-	   $(SRCDIR)/median.c \
-	   $(SRCDIR)/mload.c \
-	   $(SRCDIR)/onset.c \
-	   $(SRCDIR)/pitsyn.c \
-	   $(SRCDIR)/placea.c \
-	   $(SRCDIR)/placev.c \
-	   $(SRCDIR)/preemp.c \
-	   $(SRCDIR)/prepro.c \
-	   $(SRCDIR)/random.c \
-	   $(SRCDIR)/rcchk.c \
-	   $(SRCDIR)/synths.c \
-	   $(SRCDIR)/tbdm.c \
-	   $(SRCDIR)/voicin.c \
-	   $(SRCDIR)/vparms.c \
+SRCS	+= lpc_10_codec.c
+EXTRALIBS = -llpc10
 
 vpath	%.o $(OBJDIR)
 vpath	%.c $(SRCDIR)
--- openh323-1.15.3/plugins/audio/Speex/speexcodec.cxx.orig	2005-02-07 01:41:32.000000000 +0100
+++ openh323-1.15.3/plugins/audio/Speex/speexcodec.cxx	2005-03-25 22:24:08.087250680 +0100
@@ -82,7 +82,7 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include "libspeex/speex.h" 
+#include <speex/speex.h> 
 
 #define NARROW_SAMPLES_PER_FRAME       160
 #define WIDE_SAMPLES_PER_FRAME         320
