--- src/speexcodec.cxx.orig	2004-01-26 07:27:10.000000000 +0100
+++ src/speexcodec.cxx	2004-01-26 07:32:19.000000000 +0100
@@ -560,7 +560,7 @@
 
   // encode PCM data in sampleBuffer to buffer
   speex_bits_reset(bits); 
-  speex_encode(coder_state, floatData, bits); 
+  speex_encode(coder_state, (short int *)floatData, bits); 
 
   length = speex_bits_write(bits, (char *)buffer, encoder_frame_size); 
 
@@ -574,7 +574,7 @@
 
   // decode Speex data to floats
   speex_bits_read_from(bits, (char *)buffer, length); 
-  speex_decode(coder_state, bits, floatData); 
+  speex_decode(coder_state, bits, (short int *)floatData); 
 
   // convert float to PCM
   PINDEX i;
