--- openh323/configure.in.orig	2003-09-03 22:18:11.000000000 +0200
+++ openh323/configure.in	2003-09-03 22:19:44.977214368 +0200
@@ -45,25 +45,15 @@
   AC_CHECK_FILE(${PWLIBDIR}/version.h, HAS_PTLIB=1)
 fi
 if test "${HAS_PTLIB:-unset}" = "unset" ; then
-  AC_CHECK_FILE(${OPENH323DIR}/../pwlib/version.h, HAS_PTLIB=1)
-  if test "${HAS_PTLIB:-unset}" != "unset" ; then
-    PWLIBDIR="${OPENH323DIR}/../pwlib"
-  else
-    AC_CHECK_FILE(${HOME}/pwlib/include/ptlib.h, HAS_PTLIB=1)
-    if test "${HAS_PTLIB:-unset}" != "unset" ; then
-      PWLIBDIR="${HOME}/pwlib"
-    else
       AC_CHECK_FILE(/usr/local/include/ptlib.h, HAS_PTLIB=1)
       if test "${HAS_PTLIB:-unset}" != "unset" ; then
         AC_PATH_PROG(PTLIB_CONFIG, ptlib-config, , /usr/local/bin)
       else
         AC_CHECK_FILE(/usr/include/ptlib.h, HAS_PTLIB=1)
         if test "${HAS_PTLIB:-unset}" != "unset" ; then
-          AC_PATH_PROG(PTLIB_CONFIG, ptlib-config, , /usr/share/pwlib/make/)
+          AC_PATH_PROG(PTLIB_CONFIG, ptlib-config, , /usr/bin:/usr/share/pwlib/make)
         fi
       fi
-    fi
-  fi
 fi
 
 if test "${HAS_PTLIB:-unset}" = "unset" ; then
