diff -durN openh323.orig/openh323u.mak openh323/openh323u.mak
--- openh323.orig/openh323u.mak	Mon Oct 30 01:23:56 2000
+++ openh323/openh323u.mak	Thu Jan 11 10:20:46 2001
@@ -32,7 +32,7 @@
 endif
 
 ifndef PWLIB_FILE
-include $(PWLIBDIR)/make/ptlib.mak
+include $(PWLIBDIR)/share/pwlib/ptlib.mak
 endif
 
 
@@ -41,7 +41,7 @@
 OH323_LIBDIR = $(OPENH323DIR)/lib
 OH323_OBJDIR = $(OH323_LIBDIR)/$(PT_OBJBASE)
 
-OH323_BASE  = h323_$(PLATFORM_TYPE)_$(OBJ_SUFFIX)
+OH323_BASE  = h323
 OH323_FILE  = lib$(OH323_BASE)$(LIB_TYPE).$(LIB_SUFFIX)
 
 LDFLAGS	    += -L$(OH323_LIBDIR)
@@ -95,6 +95,7 @@
 
 $(OBJS) :	$(OH323_LIBDIR)/$(OH323_FILE)
 
+ifdef OPENH323_BUILD
 ifndef OD323_LIBRARY_MAKEFILE
 ifdef DEBUG
 $(OH323_LIBDIR)/$(OH323_FILE):
@@ -104,7 +105,7 @@
 	$(MAKE) -C $(OH323_SRCDIR) opt
 endif
 endif
-
+endif
 
 # End of file
 
diff -durN openh323.orig/src/Makefile openh323/src/Makefile
--- openh323.orig/src/Makefile	Thu Jan 11 10:19:02 2001
+++ openh323/src/Makefile	Thu Jan 11 10:19:27 2001
@@ -110,7 +110,7 @@
 PWLIBDIR=$(HOME)/pwlib
 endif
 
-include $(PWLIBDIR)/make/unix.mak
+include $(PWLIBDIR)/share/pwlib/unix.mak
 
 ifndef OPENH323DIR
 OPENH323DIR=$(HOME)/openh323
@@ -330,13 +330,13 @@
 
 ####################################################
 
-include $(PWLIBDIR)/make/common.mak
+include $(PWLIBDIR)/share/pwlib/common.mak
 
 ####################################################
 
 LIB_BASENAME	=	$(OH323_FILE)
 
-include $(PWLIBDIR)/make/lib.mak
+include $(PWLIBDIR)/share/pwlib/lib.mak
 
 ####################################################
 
