--- openh323-1.15.3/configure.ac.orig	2005-03-25 22:26:40.667055000 +0100
+++ openh323-1.15.3/configure.ac	2005-03-25 22:31:03.564088600 +0100
@@ -210,14 +210,14 @@
     fi
 
     if test "${H323_AVCODEC:-unset}" != "unset" ; then
-      AC_CHECK_FILE(/usr/local/include/ffmpeg/avcodec.h, H323_AVCODEC=1)
+      AC_CHECK_HEADER([ffmpeg/avcodec.h], H323_AVCODEC=1)
     fi
 
     if test "${H323_AVCODEC:-unset}" != "unset" ; then
       AC_SUBST(H323_AVCODEC, 1)
       AC_DEFINE(H323_AVCODEC, 1, "ffmpeg based avodec found")
-      AVCODECINCLUDEDIR=/usr/local/include/ffmpeg
-      AVCODECLIBDIR=/usr/local/lib
+      AVCODECINCLUDEDIR=/usr/include
+      AVCODECLIBDIR=/usr/lib
       STDCCFLAGS="$STDCCFLAGS -I${AVCODECINCLUDEDIR}"
 
       echo  "                      H263 codec on FFmpeg library  enabled"
--- openh323/src/ffh263codec.cxx.orig	2003-06-06 08:32:08.000000000 +0200
+++ openh323/src/ffh263codec.cxx	2003-07-13 01:10:23.000000000 +0200
@@ -109,7 +109,7 @@
 #include "rtp.h"
 
 extern "C" {
-#include <avcodec.h>
+#include <ffmpeg/avcodec.h>
 };
 
 
