diff -urN openh323.org/src/Makefile openh323/src/Makefile
--- openh323.org/src/Makefile	Tue Jan  8 00:59:12 2002
+++ openh323/src/Makefile	Tue Jan  8 01:03:10 2002
@@ -538,7 +538,7 @@
 # If we're cross compiling, we want the host's asnparser
 # otherwise use the one for the current platform
 ASNPARSE_DIR = $(HOSTPWLIBDIR)/tools/asnparser
-ASNPARSER = $(ASNPARSE_DIR)/obj_$(HOST_PLATFORM_TYPE)_d/asnparser
+ASNPARSER = $(PWLIBDIR)/bin/asnparser
 
 
 # If not cross compiling then make sure asnparser is built
